gulpfile.js

```diff
@@ -318,6 +318,17 @@ gulp.task('connect', ['watch'], function() {
     res.set('Content-Type', 'application/javascript');
     res.send(configJs);
   });
+  //TODO check up with jdyson to figure out how this works on prod
+  hawtio.use('/apiman/config.js', function(req, res, next) {
+    var configJs = 'window.APIMAN_CONFIG = {' +
+      ' auth: {' +
```

Comment:
If you use $http for your requests then hawtio-oauth automatically adds the
token as a header, so you should be all set there, at least client-side...

On Thu, Apr 30, 2015 at 9:30 AM Kurt Stam <notifications@github.com> wrote:

> In gulpfile.js
> <https://github.com/fabric8io/fabric8-console/pull/1#discussion_r29428763>
> :
>
> > @@ -318,6 +318,17 @@ gulp.task('connect', ['watch'], function() {
> >      res.set('Content-Type', 'application/javascript');
> >      res.send(configJs);
> >    });
> > +  //TODO check up with jdyson to figure out how this works on prod
> > +  hawtio.use('/apiman/config.js', function(req, res, next) {
> > +    var configJs = 'window.APIMAN_CONFIG = {' +
> > +      ' auth: {' +
>
> You tell me :). I want it to participate in the SSO session of the hawtio
> console.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fabric8io/fabric8-console/pull/1/files#r29428763>.
>